packages/tabs/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@availity/mui-tabs",
+  "version": "0.0.0",
+  "description": "Availity MUI Tabs Component - part of the @availity/element design system",
+  "keywords": [
+    "react",
+    "typescript",
+    "availity",
+    "mui"
+  ],
+  "homepage": "https://availity.github.io/element/?path=/docs/components-tabs-introduction--docs",
+  "bugs": {
+    "url": "https://github.com/Availity/element/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Availity/element.git",
+    "directory": "packages/tabs"
+  },
+  "license": "MIT",
+  "author": "Availity Developers <AVOSS@availity.com>",
+  "browser": "./dist/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --dts",
+    "dev": "tsup src/index.ts --format esm,cjs --watch --dts",
+    "clean": "rm -rf dist",
+    "clean:nm": "rm -rf node_modules",
+    "bundlesize": "bundlesize",
+    "publish": "yarn npm publish --tolerate-republish --access public",
+    "publish:canary": "yarn npm publish --access public --tag canary"
+  },
+  "dependencies": {
+    "@mui/lab": "^5.0.0-alpha.121"
```

Comment:
@LauRoxx This should probably be a devDep and peerDep, right?

Reply:
We list material as a peerdep because we have it as a full dependency for element and didn't want mismatched versions across all the packages. we could definitely go that way for lab as well, but right now it's only loadingbutton and tab that uses it so I could see it going either way. up to you.

Reply:
Ok, thanks for the clarification!